src/diffusers/models/unet_2d_condition.py

```diff
@@ -528,6 +528,30 @@ def fn_recursive_add_processors(name: str, module: torch.nn.Module, processors:
 
         return processors
 
+    @property
```

Comment:
This is identical to:

```py
out = {k + "." + in_k: in_v for k, in_d in  pipe.unet.attn_processors.items() for in_k, in_v in in_d.state_dict().items()
```

no? 

I'd maybe prefer to not add a new property and instead retrieve this values with the help of something like

```py
out = {k + "." + in_k: in_v for k, in_d in  pipe.unet.attn_processors.items() for in_k, in_v in in_d.state_dict().items()
```

Reply:
yeah can change